icu4c/source/common/brkeng.h

```diff
@@ -80,6 +83,35 @@ class LanguageBreakEngine : public UMemory {
 
 };
 
+/*******************************************************************
+ * ExternalLanguageBreakEngine
+ */
+
+/**
+ * <p>ExternalLanguageBreakEngine implement LanguageBreakEngine by 
+ * a thin wrapper that delegate the task to ExternalLanguageBreakEngine
```

Comment:
Should the second `ExternalLanguageBreakEngine` in this comment be `ExternalBreakEngine`?  That's what the code below seems to be saying.

Also, the naming here is kind of confusing: You have `LanguageBreakEngine`, `ExternalBreakEngine`, and `ExternalLanguageBreakEngine`, and it's not clear what their relationship is to one another.